scripts/gitlab/build-unix.sh

```diff
@@ -11,6 +11,11 @@ echo "CC:               " $CC
 echo "CXX:              " $CXX
 #strip ON
 export RUSTFLAGS=" -C link-arg=-s"
+# Linker for crosscomile
+echo "[target.$CARGO_TARGET]" >> .cargo/config
+echo "linker= \"$CC\"" >> .cargo/config
```

Comment:
config file will grow once caching is back enabled.

Reply:
we will not use caching for this task. it is not necessary

Reply:
fixed